### PR TITLE
ci: increase test timeout

### PR DESCRIPTION
**Pull request by timdeschryver**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Other information

All of our builds are suddenly failing.
After some research it seems due to a timeout during the CI build.

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -82,6 +82,7 @@ jobs:
       - run:
           name: Run All Unit Tests
           command: yarn nx run-many -t test
+          no_output_timeout: 20m
```
Comment (timdeschryver) [inline .circleci/config.yml:85]: The default is 10 minutes, and it seems like we're just surpassing it with 10m 25s...

File: .circleci/config.yml [hunk 2]
```diff
@@ -10,7 +10,7 @@ version: 2.1
 # See https://blog.daemonl.com/2016/02/yaml.html
 # To validate changes, use an online parser, eg.
 # https://yaml-online-parser.appspot.com/
-var_1: &cache_key yarn-cache-{{ checksum "yarn.lock" }}-0.18.0
+var_1: &cache_key yarn-cache-{{ checksum "yarn.lock" }}-0.18.1
```
Comment (rainerhahnekamp) [inline .circleci/config.yml:13]: Out of interest, why do we have the suffix 0.18.1? Is the checksum not already enough?

File: .circleci/config.yml (hunk 1 above)
Comment (rainerhahnekamp) [reply .circleci/config.yml:85]: Yeah, and we already see that the build took more than twelve minutes and was successful. So it works. 👍

File: .circleci/config.yml (hunk 2 above)
Comment (timdeschryver) [reply .circleci/config.yml:13]: The checksum should be enough, and was in this case not the culprit.
In the past we've had some experiences where the cache could go corrupt causing the CI build to act funky. That's why my first idea was to create a new cache.

**Review by rainerhahnekamp — APPROVED** [review]

**Review by markostanimirovic — APPROVED** [review]

**Pull request merged**